### PR TITLE
[READY] Vago Class Light Missile Cruiser

### PR DESCRIPTION
**Pull request by Vasily2013**
## About The Pull Request

This PR introduces Vago, a small to medium size torpedo bus that is primarily intended for player numbers around 20 people.

>INCOMING PRIORITY TRANSMISSION FROM NANOTRASEN HEADQUARTERS
>DUE TO INCREASED ENEMY ACTIVITY A NEW SHIP CLASS IS BEING DISPATCHED INTO OPERATIONS. BE ADVISED THIS IS AN EXPERIMENTAL SHIP.
>FEATURES AS FOLLOWS:
-Convenient, centralized command systems to increase ease of use for operators
-Condensed form factor to reduce crew fatigue and increase productivity
-Exclusively unique bar that also serves as a primary evacuation point for the ship
-A significant increase in vertical capability compared to the Aegis class flag ships
-Increased number of airlocks per-cubic meter of internal volume
-Reduced retrofit time in drydock

## Why It's Good For The Game

Aegis needs an ass to front rework, and another ship might be good.

## Changelog
:cl:
add: A new map named Vago
edit: changed a bug on Valkyrie that would have had miners starting with golum equipment vendor
/:cl:

image
image

### CODE REVIEW

**Comment by Vasily2013** [comment]
NOTE: this map still requires overmap things. I'm going to leave the overmap BS to KMC.

File: config/maps.txt [hunk 1]
```diff
@@ -33,3 +33,6 @@ endmap
 
 map multiz_debug
 endmap
+
+map Vago
```
Comment (francinum) [inline config/maps.txt]: Move this above the debug maps

Comment (Vasily2013) [reply config/maps.txt]: donezo

**Comment by Kmc2000** [comment]
Can you change the stock ss13 windows to the new spicy ones i just added?

**Comment by Kmc2000** [comment]
Is this ready for overmap memes

**Comment by Vasily2013** [comment]
> Bridge door leading to a shuttle but also space. Intentional?
> Railgun facing the wrong way in munitions (flip it)
> Cafeteria area is a little grey, can you spice it up with some turf decals?
> 
> Apart from that, this map looks pretty nice actually

-Centcom door, intentional
-I'll flip the railgun when you change the default sprite to point the right way on ALL OF OUR EAST FACING MAPS.
-I'll look into this in the not to distant future. In the meantime feel free to visit the escape bar.

File: code/datums/map_config.dm [hunk 2]
```diff
@@ -107,7 +107,7 @@
 		// traits you want to customize which level is cross-linked
 		for (var/level in traits)
 			if (!(ZTRAIT_STATION in level))
```
Comment (Jalleo) [inline code/datums/map_config.dm]: This needs the change too

Comment (Vasily2013) [reply code/datums/map_config.dm]: fixed

**Comment by Wilchenx** [comment]
Its a rectangle and is named like 70's soviet cars

Perfect.

**Comment by Vasily2013** [comment]
Ready for Merge pending reviews.

**Comment by Cenrus** [comment]
torp bus torp bus

**Pull request merged**